README.md

```diff
@@ -376,5 +376,6 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | :------ | :---------- | :-------- |
 | [eDreams ODIGEO](https://www.edreamsodigeo.com/) [:rocket:](https://www.edreamsodigeocareers.com) | We are shaping the way people travel today and tomorrow. | `Porto` |
 | [Hostelworld](https://www.hostelworld.com/) [:rocket:](https://www.hostelworldgroup.com/careers) | World's leading hostel-focused online booking platform. | `Porto` |
+| [Ryanair](https://www.ryanair.com/pt/pt) [:rocket:](https://careers.ryanair.com) | Ryanair is an Irish ultra low-cost carrier. | `Lisboa` `Porto`|
```

Comment:
Maybe we should use this link instead (https://careers.ryanair.com/search/#search/location=portugal&departments=ryanair-labs-portugal). since it points directly to vacancies for the Portugal Labs department, which is the one hiring tech employees.